src/commands/apps-android-sha-list.ts

```diff
@@ -0,0 +1,59 @@
+import * as clc from "cli-color";
+import * as ora from "ora";
+import Table = require("cli-table");
+
+import { Command } from "../command";
+import * as getProjectId from "../getProjectId";
+import { listAppAndroidSha, AppAndroidShaData } from "../management/apps";
+import { requireAuth } from "../requireAuth";
+import * as logger from "../logger";
+
+function logCertificatesList(certificates: AppAndroidShaData[]): void {
+  if (certificates.length === 0) {
+    logger.info(clc.bold("No SHA certificate hashes found."));
```

Comment:
Normally we don't bold the entire message, suggest just removing the bold part here.